### PR TITLE
Upgrade to dozer v6.4.0 #829

### PR DESCRIPTION
**Pull request by btshuugna**
Please review #829
Change dozer version to v6.4.0

### CODE REVIEW

File: terasoluna-gfw-parent/pom.xml [hunk 1]
```diff
@@ -397,7 +398,7 @@
     <org.mybatis.version>3.4.6</org.mybatis.version>
     <org.mybatis.spring.version>1.3.2</org.mybatis.spring.version>
     <!-- == Dozer== -->
-    <dozer.version>6.2.0</dozer.version>
+    <dozer.version>6.3.0</dozer.version>
```
Comment (yoshikawaa) [inline terasoluna-gfw-parent/pom.xml]: - [ ] Dozer 6.4.0 has bean released. Please re-consider using version.

File: terasoluna-gfw-parent/pom.xml [hunk 2]
```diff
@@ -211,6 +211,7 @@
         <groupId>com.github.dozermapper</groupId>
         <artifactId>dozer-core</artifactId>
         <version>${dozer.version}</version>
+        <classifier>jar-with-dependencies</classifier>
```
Comment (yoshikawaa) [inline terasoluna-gfw-parent/pom.xml]: - [ ] Why required `classifier=jar-with-dependencies`? Is there no problem even without it?

File: terasoluna-gfw-parent/pom.xml (hunk 1 above)
Comment (btshuugna) [reply terasoluna-gfw-parent/pom.xml]: OK,I'll upgrade Dozer's version to 6.4.0.Thank you.

File: terasoluna-gfw-parent/pom.xml [hunk 3]
```diff
@@ -398,7 +397,7 @@
     <org.mybatis.version>3.4.6</org.mybatis.version>
     <org.mybatis.spring.version>1.3.2</org.mybatis.spring.version>
     <!-- == Dozer== -->
-    <dozer.version>6.3.0</dozer.version>
+    <dozer.version>6.4.0</dozer.version>
```
Comment (yoshikawaa) [inline terasoluna-gfw-parent/pom.xml]: @btshuugna 
- [x] I checked them.
* [6.4 release note](https://github.com/DozerMapper/dozer/releases/tag/6.4.0)
* [6.3 to 6.4 migration guide](https://github.com/DozerMapper/dozer/blob/master/docs/asciidoc/migration/v63-to-v64.adoc)

And understand it's no problem to upgrade.
But it seems to need reflect follows topic to our guideline.
* Java 8 date and time support

**Pull request merged**